modules/tensor_mechanics/src/materials/StressUpdateBase.C

```diff
@@ -31,3 +31,8 @@ StressUpdateBase::setQp(unsigned int qp)
 {
   _qp = qp;
 }
+
+void
```

Comment:
I think this function should be pure virtual, and we shouldn't provide an empty implementation in the base class.